src/Classes/Options/Particles/OpacityAnimation.ts

```diff
@@ -15,6 +18,8 @@ export class OpacityAnimation implements IOpacityAnimation {
      * @param value
      */
     public set opacity_min(value: number) {
+        Messages.deprecationMessage("particles.opacity.animation.opacity_min", "particles.opacity.animation.minimumValue");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139697)